src/dashboard/Settings/components/SettingsLayout/index.js

```diff
@@ -0,0 +1,119 @@
+import classnames from 'classnames';
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Button, Col, Input, Row, Layout, Checkbox } from 'antd';
+import PropTypes from 'prop-types';
+import SidebarSettings from '../SidebarSettings';
+import ColHeader from '../../../../common/components/ColHeader';
+import IncidentType from '../SystemSettings/Components/IncidentTypeList';
+import IncidentDetails from '../SystemSettings/Components/IncidentDetails';
+import AddNewIncidentType from '../SystemSettings/Components/IncidentTypeForm/AddNewIncidentType';
+import EditIncidentTypeForm from '../SystemSettings/Components/IncidentTypeForm/EditIncidentTypeForm';
+
+/* load styles */
+import styles from './styles.css';
+
+/* local constants */
+const { Content } = Layout;
+const { Search } = Input;
+const cx = classnames.bind(styles);
+
+/**
+ * Settings layout display
+ *
+ * @function
+ * @name SettingsLayout
+ * @param {Object} props
+ * @param {string} props.incidentType
+ *
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+
+const SettingsLayout = props => {
```

Comment:
For component please name the function don't use arrow functions since it is hard to know the name of the component if it throws errors

```js
function SettingsLayout(){}
```

Reply:
Also I suggest destructing the props so it will be easy to document what props it receives